application/src/test/java/uk/gov/hmcts/net/ccd/definition/store/BaseTest.java

```diff
@@ -157,10 +162,11 @@ protected DisplayItemsData mapDisplayItemsData(ResultSet resultSet, Integer i) t
 
     protected void assertResponseCode(MvcResult mvcResult, int httpResponseCode) throws UnsupportedEncodingException {
         MockHttpServletResponse response = mvcResult.getResponse();
-        assertEquals("Expected [" + httpResponseCode + "] but was [" + response.getStatus() + "]"
-                + " Body was [\n" + response.getContentAsString() + "\n]",
+        assertEquals(
             httpResponseCode,
-            mvcResult.getResponse().getStatus()
+            mvcResult.getResponse().getStatus(),
```

Comment:
Could use response.getStatus() here instead of mvcResult.getResponse().getStatus().